src/homework/day3/homework3.java

```diff
@@ -0,0 +1,40 @@
+package homework.day3;
+
+import java.util.Scanner;
+
+public class homework3 {
+    public static int lowest(int[] numbers) {
+        int lowest = numbers[0];
+        for (int i = 1; i < numbers.length; i++) {
+            if (lowest > numbers[i]) {
+                lowest = numbers[i];
+            }
+        }
+        return lowest;
+    }
+
+    public static void trueOrFalse(int number) {
```

Comment:
Method's name is not explaining what it does. Rename it better. For example: checkIfNumberIsOdd or checkIfNumberIsEven.